manifests/volume_group.pp

```diff
@@ -1,4 +1,19 @@
 # == Define: lvm::volume_group
+# @param physical_volumes The list of physical volumes to be included in the volume group; 
+# this will automatically set these as dependencies, but they must be defined elsewhere using the physical_volume resource type.
+
+# @param createonly Default value: false - If set to true the volume group will be created if it does not exist. 
```

Comment:
puppet-strings will pick up the datatype and default value from the code, so you dont need to list it here.

Reply:
Updated